src/debugger.ts

```diff
@@ -172,6 +176,7 @@ export namespace Debugger {
     private _session: IDebugger.ISession | null;
     private _sessionChanged = new Signal<this, void>(this);
     private _service = new DebugService(null, this);
+    private _selectCurrentLine = new Signal<this, number>(this);
```

Comment:
How about this name to follow the other conventions?

```suggestion
    private _currentLineChanged = new Signal<this, number>(this);
```